pkg/bootkube/create.go

```diff
@@ -23,40 +23,34 @@ func CreateAssets(manifestDir string, timeout time.Duration) error {
 
 	upFn := func() (bool, error) {
 		if err := apiTest(); err != nil {
-			glog.Warningf("Unable to determine api-server version: %v", err)
+			glog.Warningf("Unable to determine api-server readiness: %v", err)
 			return false, nil
 		}
 		return true, nil
 	}
 
 	createFn := func() (bool, error) {
 		err := createAssets(manifestDir)
-		if err != nil {
-			glog.Errorf("Error creating assets: %v", err)
-			// If error is "system namespace not found" we should retry
-			if apierrors.IsNotFound(err) {
-				details := err.(*apierrors.StatusError).Status().Details
-				if details.Name == api.NamespaceSystem && details.Kind == "namespaces" {
-					return false, nil // retry
-				}
-			}
-			UserOutput("\nError creating assets: %v\n", err)
-			UserOutput("\nNOTE: Bootkube failed to create some cluster assets. It is important that manifest errors are resolved and resubmitted to the apiserver.\n")
-			UserOutput("For example, after resolving issues: kubectl create -f <failed-manifest>\n\n")
-		}
-		return true, nil
+		return true, err
```

Comment:
Thanks for the background! It wasn't my intent to change the behavior. I'll go back to returning nil and add a comment explaining why.